releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -1983,13 +2064,12 @@
     "sigs": [
       "api-machinery"
     ],
-    "duplicate_kind": true,
-    "is_mapped": true
+    "duplicate_kind": true
   },
   "124557": {
     "commit": "ae8474adcdb5957ebaef82de55fbabdf2433356a",
     "text": "The nftables kube-proxy mode now has its own metrics rather than reporting\nmetrics with \"iptables\" in their names.",
-    "markdown": "The nftables kube-proxy mode now has its own metrics rather than reporting\n  metrics with \"iptables\" in their names. ([#124557](https://github.com/kubernetes/kubernetes/pull/124557), [@danwinship](https://github.com/danwinship)) [SIG Network and Windows]",
+    "markdown": "The nftables kube-proxy mode now has its own metrics rather than reporting\n  metrics with \"iptables\" in their names. ([#124557](https://github.com/kubernetes/kubernetes/pull/124557), [@danwinship](https://github.com/danwinship))",
```

Comment:
```suggestion
    "markdown": "The nftables kube-proxy mode now has its own metrics rather than reporting\n  metrics with \"iptables\" in their names. ([#124557](https://github.com/kubernetes/kubernetes/pull/124557), [@danwinship](https://github.com/danwinship)) [SIG Network and Windows]",
```